### PR TITLE
additional strings and changelog

### PR DESCRIPTION
**Pull request by cliffordp**
[ETP-145]

Related: https://github.com/moderntribe/event-tickets-plus/pull/949

[ETP-145]: https://moderntribe.atlassian.net/browse/ETP-145

### CODE REVIEW

File: src/admin-views/tpp-metabox-sku.php [hunk 1]
```diff
@@ -27,6 +27,10 @@ class="ticket_field sku_input ticket_form_right"
 		value="<?php echo esc_attr( $sku ); ?>"
 	>
 	<p class="description ticket_form_right">
-		<?php esc_html_e( 'A unique identifying code for each ticket type you\'re selling', 'event-tickets' ); ?>
+		<?php echo esc_html( sprintf(
```
Comment (jesseeproductions) [inline src/admin-views/tpp-metabox-sku.php]: I could not find a note in the standards, but check the coding and where you have a opening <?php 
Lets be consistent and  start the code on the next line or right after. 
```
<?php
echo esc_html(
)
?>
```
or

```
<?php echo esc_html(
)
?>
```

Inline PHP would remain the same.

File: src/admin-views/editor/fieldset/settings-provider.php [hunk 2]
```diff
@@ -29,8 +29,25 @@ class="tribe-ticket-editor-field-default_provider settings_field"
 			<?php endforeach; ?>
 		<?php else : ?>
 			<section style="margin-bottom: 0;">
-				<legend id="default_ticket_provider_legend" class="ticket_form_left"><?php esc_html_e( 'Sell tickets using:', 'event-tickets' ); ?></legend>
-				<p class="ticket_form_right"><?php esc_attr_e( 'It looks like you have multiple ecommerce plugins active. We recommend running only one at a time. However, if you need to run multiple, please select which one to use to sell tickets for this event.' ); ?> <em><?php esc_attr_e( 'Note: adjusting this setting will only impact new tickets. Existing tickets will not change. We highly recommend that all tickets for one event use the same ecommerce plugin.', 'event-tickets' ); ?></em></p>
+				<legend id="default_ticket_provider_legend" class="ticket_form_left"><?php
+					echo esc_html( sprintf(
```
Comment (jesseeproductions) [inline src/admin-views/editor/fieldset/settings-provider.php]: Putting the <?php and ?>  on their own lines could make this a bit more readable.

Comment (cliffordp) [reply src/admin-views/editor/fieldset/settings-provider.php]: done

File: src/admin-views/tpp-metabox-sku.php (hunk 1 above)
Comment (cliffordp) [reply src/admin-views/tpp-metabox-sku.php]: done

**Comment by Camwyn** [comment]
Hold off until we get the tests fixed?

**Pull request merged**